Makefile

```diff
@@ -14,9 +14,16 @@ GIT_DIRTY = `git status --porcelain`
 GIT_TAG = `git describe --tags || echo "no version info"`
 AUTHOR = $(USER)
 
+
+# Test report configuration
+TEST_REPORT_DIR ?= $(CURRENT_DIR)/tests/report
+TEST_REPORT_FILE ?= nose2-junit.xml
```

Comment:
Why adding test report in makefile but not in CI ? Moreover we stopped using this report in CI because they didn't bring any plus value.